config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: gcr.io/k8s-staging-nfd/node-feature-discovery-operator
-  newTag: master
+  newName: registry.k8s.io/nfd/node-feature-discovery-operator
+  newTag: 0.4.2
```

Comment:
I'd suggest to leave this as it was

Reply:
good catch. This gets set by the `make deploy` process so they keep creeping in while I'm testing, I obviously missed this one and committed it by mistake. I have reverted it.